packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.js

```diff
@@ -114,6 +113,9 @@ export default class FocusManager extends Base {
       x: this.itemPosX,
       y: this.itemPosY
     });
+    if (this._lazyItems) {
+      this._lazyItems = null;
```

Comment:
does it matter that this is null instead of an empty array

Reply:
Yeah, this is actually broken on develop now too. NavigationManager's `appendItems()` method does this check, so if that array exists when set items is called they just get pushed there and not rendered.

```
  appendItems(items = []) {
    this.shouldSmooth = false;

    if (this._lazyItems) {
      this._lazyItems.push(...items);
      return;
    }
 ```

Reply:
*will handle error checking for this in a separate ticket